compiler/noirc_evaluator/src/brillig/brillig_gen/constant_allocation.rs

```diff
@@ -165,62 +177,12 @@ impl ConstantAllocation {
         current_block
     }
 
+    /// Return the SSA [ValueId] of all constants (they might have the same value).
```

Comment:
By "same value" do you mean same numeric constant? I think that would be clearer as to not mess with SSA value terminology.

Reply:
Yes, there was a comment somewhere that this is possible, although I wasn't sure exactly how. I'll rephrase.